integration-tests/amazon-cloudwatch-observability/validator/validateResources_test.go

```diff
@@ -1,6 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build linuxonly || windowslinux
```

Comment:
Do we need this?

Reply:
Not sure, but this is how it was in the previous package I migrated from: https://github.com/aws/amazon-cloudwatch-agent-operator/blob/cab619e55a8719dbfcf7756d0e1647f9d7834521/integration-tests/eks/validateResources_test.go#L4.

I know it was in the other packages to associate tags with it.

Reply:
Yeah that's needed coz those tags are used to distinguish linux/windows integration tests